README.md

```diff
@@ -27,6 +27,7 @@ service/environment.
   - **true**: Disables the ZipkinMiddleware/ZipkinMessageHandler
 - `ZipkinBaseUri` - is the zipkin scribe/collector server URI with port to send the Spans
 - `Domain` - is a valid public facing base url for your app instance. Zipkin will use to label the trace.
+  - by default this looks at the incoming requests and uses the hostname from them. It's a `Func<IOwinRequest, Uri>` - customise this to your requirements.
```

Comment:
@petemounce Please update the sample below on instantiating the ZipkinConfig with regards to this new change. Thanks